### ISSUE TITLE
OPS: Minimal Wording Changes for Emphasis on Preprints

### ISSUE THREAD

**Issue by willinsky** [pkp/pkp-lib#5610]
**Describe the bug**
The following changes are intended to distinguish "preprints" from "publications" which is an important part of the preprint's place in the scholarly communication.

A. https://demo.publicknowledgeproject.org/ops/testdrive/index.php/publicknowledge/workflow/index/17/5

1. Change the tab “Publication” to “Preprint”
2. Change the button “Schedule For Publication” to “Post the preprint”
3. Change “Status” from “Unscheduled” and “Published” to “Unposted” and “Posted”; and similarly replace “published” with “posted” in related steps. 
4. On the pop-up called “Publish,” which reads as follows…. 
Publish
All publication requirements have been met. Are you sure you want to publish this?
									[Publish]
CHANGE TO...
Post 
All preprint requirements have been met. To post it, click the button below.
								    	[Post]

B.
https://demo.publicknowledgeproject.org/ops/testdrive/index.php/publicknowledge/workflow/index/17/5
1. Change “Production Discussions” to “Preprint Discussions”
2. Change the logo “Platform & Workflow PPS / PKP” to the logo “Platform & workflow OPS / PKP”  which is found here:
https://practical-chandrasekhar-e6ae6f.netlify.com/ 
3. The logo appears too large on the public facing side of the site. "Platform & Workflow" should be the same size as the word "Search" at the top of the page. 

C.
https://demo.publicknowledgeproject.org/ops/testdrive/index.php/publicknowledge/preprint/view/26 
1. Change "Published" to "Posted"

**Comment by NateWr** [comment]
@alexxxmendonca does Scielo have a view on whether this distinction is important to make in the workflow?

**Comment by alexxxmendonca** [comment]
@NateWr we discussed this on our last meeting with PKP (which took place on March 5).

We agree with @willinsky's suggestions and I remember there was also some discussion around the Editor role.

I don't seem to remember us having reach any conclusion but my suggestion would be to change that role name to "Moderator".

**Comment by NateWr** [comment]
Thanks! @ajnyga let me know if you have any questions.

**Comment by willinsky** [comment]
Thanks Nate for bringing Alex into this. And thanks Alex for the "moderator" suggestion, as this role would map well onto OJS' editor. The role is used, as  you likely know, by [arxiv.org](https://arxiv.org/help/moderation) to check the basic legitimacy of every submission, and if we follow this model the moderators could be assigned to categories of preprints, notified as each one comes in, and is the one, after a screening, to push "Post the preprint" button. (On the other hand, [bioRxiv.org](https://www.biorxiv.org/about-biorxiv) calls on "Affiliates" on occasion while having staff (=OPS managers) screen every submission for the basics.) We could include the "moderator" role in OPS as an editor-like role, with the OPS manager assigning moderators, while the manager fills the role until a network of moderators is established, much as arXiv.org has (and probably took years to establish). While this differs from our earlier discussions with Abel around automated approval of qualified authors, without discussing further screening, I can see having, further down the road, an option for an Unmoderated OPS, with the Moderated the default.

**Comment by alexxxmendonca** [comment]
@willinsky agreed 👍

**Comment by asmecher** [comment]
The discussion of the Moderator role is filed over here: https://github.com/pkp/pkp-lib/issues/5622

**Comment by ajnyga** [comment]
Basically almost all of these translations come from the shared library. Besides the examples mentioned above there are also a lot of other places where the words "publication" and "publish" appear.

These are rough numbers, but these translation keys live in `api.po` (5 strings), `common.po` (5 strings), `editor.po` (1 string), `submissions.po` (25 strings), `user.po` (1 string).

@asmecher @NateWr should I 1) override the shared lib keys in OPS or 2) move the above keys to the application specific translation files first (meaning that I first edit all existing translations for OJS and OMP). Of course the latter is "correct" way of doing it, but just want to make sure you agree before I start while it is a lot of work.

**Comment by NateWr** [comment]
My feeling is to leave them in pkp-lib and override them in ops but Alec has a better vision for the translation stuff so wait for his recommendation.

**Comment by asmecher** [comment]
@willinsky (and @alexxxmendonca), I'm on the fence about changing "Publication" to "Posting" for OPS but leaving it as "Publication" in OJS and OMP. Is there a term that might serve for both applications? (Referring to the same thing by several terms is a frequent source of confusion, both in the application and in the source code. And our long-term goal of supporting different forms of publication within the same application will be complicated if the terminology isn't aligned.)

**Comment by willinsky** [comment]
The term "published" is more a designation of formal significance in publishing than a vocabulary choice, so I'm reluctant to move away from it for OJS or OMP. (The discussion around Covid-19 preprints is bringing this out, although the point often made is that they are not "peer-reviewed" in the context of not having been "published.") I'm not sure that this may help but let me say that where this distinction matters most is on the article landing page, where is says "Published 2020-01-13." For preprints, that is misleading for readers and could interfere with authors getting the work published elsewhere. Less important is in Production stage where the button reads "Schedule for Publication" and in the submissions queue where it lists works that are "Published." So if by chance the landing page on OPS could be more readily changed to "Posted" than the others, that would be one way to go.

**Comment by ajnyga** [comment]
Preprints.org seems to use  `Online: 15 March 2020`.

**Comment by asmecher** [comment]
Thanks, @willinsky. If it's palatable to all, let's tweak that kind of language on the front end, and continue to gather feedback and consider the back-end changes. That'll represent less divergence from OJS/OMP until we know it's justified.

**Comment by willinsky** [comment]
Okay, I can live with the public-facing changes only. I prefer "posted" as we can use it later for a post button, but I also like the Antti-Jussi's idea of checking other sites (bioArxiv.org "posted"; SSRN "posted"; arXiv.org "submitted") and open to reasons for others.

**Comment by alexxxmendonca** [comment]
I agree with @willinsky [here](https://github.com/pkp/pkp-lib/issues/5610#issuecomment-611137416) and [here](https://github.com/pkp/pkp-lib/issues/5610#issuecomment-611209446).

**Comment by ajnyga** [comment]
OPS: https://github.com/pkp/ops/pull/28

@asmecher the pr above changes the front end language. I think the only thing needing revision there was the Published > Posted change.

The pr also changes OPS towards a category centric default theme. The Section taxonomy is still available, but the theme now lists all main categories below the search field and shows the selected categories on the preprint landing page.

I also changed the tests so that only one Section is created. When the Category Editors pr is merged, I will edit the tests so that the test submissions select categories.

**Comment by NateWr** [comment]
I notice the PR above has been merged. Can this issue be closed?

**Comment by ajnyga** [comment]
It depends on how you want to track the requests concerning backend.

**Comment by NateWr** [comment]
Can you recommend an action here? Shall we bump the remaining work to the 3.3 milestone or file a new issue or do you anticipate further work here for 3.2.1?

**Comment by ajnyga** [comment]
I think it depends whether you (PKP) want to use the "Publication/Publish" related vocabulary in the backend also with OPS, or whether you want to change that. @asmecher ?

**Comment by willinsky** [comment]
If by "backend," Antti-Jussi, you are referring to the submissions dashboard and workflow, then yes, I think it important that we implement A, B, and C in the March 10th Comment above. If by "backend" you are referring to areas of code and database that managers and authors don't see, then I see no need to change. Although I see that thought is being given to the best ways to handle the exchange with the PKP common library to account for the vocabulary shift.

**Comment by willinsky** [comment]
This is a wording change that applies to the PKP Library but is particularly an acute problem for OPS because the author runs into it in the very process of uploading their submission. 

<img width="461" alt="Screen Shot 2020-11-22 at 9 03 59 PM" src="https://user-images.githubusercontent.com/10342108/99931363-0f533900-2d09-11eb-941f-1a3db1e2ca45.png">

Alternative wording for Galley popup on all systems

**Upload a Galley**
(The galley is a file that will be used to display the submission for readers)

Galley Label *
[     ]
Typically identifies the galley's file format (e.g., PDF, HTML, etc.). 
[note: remove asterisk after *, and add comma after e.g.,]

[    ]
Language

Options

[ ] Prevent the display of the galley on this website, as it will appear at another location.

URL Path [                        ]
Specify a path for the submission's URL to be used instead of the assigned ID.
[note do not bold URL Path]

**Comment by willinsky** [comment]
<img width="826" alt="Screen Shot 2020-11-22 at 10 41 50 PM" src="https://user-images.githubusercontent.com/10342108/99934811-ed12e880-2d13-11eb-81c1-3d4c823201d9.png">


Preprints cannot be treated as "archives" like past issues of journals, unless the preprint has been withdrawn or otherwise archived out of sight.  

bioRxiv.org has "Channels" that belong to different organizations, as we have "Archives" and a most elaborate advanced [search](https://www.biorxiv.org/search), but nothing like "archives." We might use "Categories," when we have them, where we have Archives but we need to remove "Archives" from that main menu line.

**Comment by asmecher** [comment]
@willinsky, about your [proposed changes](https://github.com/pkp/pkp-lib/issues/5610#issuecomment-731938021) on galley uploads...

Regarding changing...
> This galley will be available at a separate website

...to...

> Prevent galley from displaying on this website, as it will appear at another location

...this option doesn't mean that the galley won't be posted with OJS/OMP/OPS, it means that rather than uploading a file, the user can indicate a URL where the item is hosted externally.

Regarding changing...
> An optional path to use in the URL instead of the ID

...to...

> Specify a path for submission's URL to replace the assigned ID

...this is less clear to me than the original. This is a field for the user to enter something meaningful that takes the place of the default numeric ID in the URL of the posted content, in case that helps.

**Comment by willinsky** [comment]
Thank you @asmecher for clarifying and I hope that my errors make it clear how hard it has been for the likes of me to figure out the intent of the "explanations" on this particular popup. It has trouble me on encountering it for some time but as I say in this situation (with authors), it particularly calls out for clarity. Please see if I have now come to understand the two options on this popup. 

Alternative wording for Galley popup on all systems

Upload a Galley
(The galley is the file used to display the submission for readers)

Galley label *
[ ]
Identify the galley's file format (e.g., PDF, HTML, etc.).
[note: to not bold Galley Label, remove asterisk after "etc.)" and add comma after e.g.,]

Galley language *
[ ]

Options

[ ] Use the galley available at the URL entered below (after checking this box).
[Can we not just include the box to enter it (as in next option), rather than the checkbox leading to the box, and if so, drop the parenthetical part of the instruction above.]

URL Path [ ]
Replace the submission ID in the galley URL with the path provided.
[note: do not bold URL Path]

**Comment by ajnyga** [comment]
@asmecher 
Here is a pr for OPS: https://github.com/pkp/ops/pull/80

Once that is merged, I will do another pr for OJS/OMP/lib where I move those duplicated keys from the shared lib to OJS and OMP.

**Comment by ajnyga** [comment]
tests passing

**Comment by asmecher** [comment]
Thanks, @ajnyga, this looks good -- please do prepare the other PRs and I'll merge all at the same time.

**Comment by ajnyga** [comment]
@asmecher running test on all three applications.
In OJS and OMP I have moved all the duplicated keys from the shared library to the app locale files in all existing locales. In OPS I have only moved the new keys to en_US while all the other locales in OPS need to translate these.

pkp-lib: https://github.com/pkp/pkp-lib/pull/6438
OJS: https://github.com/pkp/ojs/pull/2940
OMP: https://github.com/pkp/omp/pull/894
OPS: https://github.com/pkp/ops/pull/80

**Comment by asmecher** [comment]
Those are all merged (manually to resolve conflicts) -- thanks, @ajnyga! Is this issue ready to be closed?

**Comment by ajnyga** [comment]
I think this can be closed. If something was missed (someone finds more Publish/Publication in OPS) then we can deal with those separately. The pr's I made covered everything mentioned above and also a lot of other cases not mentioned.

**Comment by asmecher** [comment]
:+1: Thanks, @ajnyga! This will go out in OJS/OMP/OPS3.3rc2 shortly.
